test/drag.spec.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ElementHandle } from '..';
+import { it, expect, describe } from './fixtures';
+import { attachFrame } from './utils';
+
+describe('Drag and drop', test => {
+  test.fixme();
+}, () => {
+  it('should work', async ({server, page, context}) => {
+    await page.goto(server.PREFIX + '/drag-n-drop.html');
+    await page.hover('#source');
+    await page.mouse.down();
+    await page.hover('#target');
+    await page.mouse.up();
+    expect(await page.$eval('#target', target => target.contains(document.querySelector('#source')))).toBe(true); // could not find source in target
```

Comment:
What does "could not find source in target" mean here?

Reply:
We use to have error messages after the error as a string, which was helpful when the test failed. When we moved to the expect library, these got removed. But the error reporter shows the line that failed now, so I just put a message in a comment after the line.